sky/packages/sky/lib/src/fn3/framework.dart

```diff
@@ -248,6 +250,18 @@ abstract class Element<T extends Widget> implements BuildContext {
   /// The configuration for this element.
   T _widget;
 
+  RenderObject get _descendantRenderObject {
+    RenderObject result;
+    void visit(Element element) {
+      if (element is RenderObjectElement)
+        result = element.renderObject;
+      else
+        visitChildren(visit);
+    }
+    visitChildren(visit);
```

Comment:
probably worth having asserts here (or somewhere) that visitChildren() for anything that isn't a RenderObjectElement only visits one child.